docs/library/overview/modules.md

```diff
@@ -1,30 +1,30 @@
 # What is a module?
 
-**Modules are reusable "infrastructure building blocks" that describe how to deploy and manage a specific piece of infrastructure, such as a VPC, ECS cluster, or Auto Scaling Group.**
+Modules are reusable "infrastructure building blocks" that describe how to deploy and manage a specific piece of infrastructure, such as a VPC, ECS cluster, or Auto Scaling Group.
```

Comment:
Same question here - did you modify this manually or did it auto regenerate for some reason?

Reply:
See https://github.com/gruntwork-io/docs/pull/1010#discussion_r1273901969.